src/Finder/ResourceFinder.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Finder;
+
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Creates a Finder object with the bundle paths set.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class ResourceFinder
+{
+    /**
+     * @var array
+     */
+    private $paths;
+
+    /**
+     * Constructor.
+     *
+     * @param string|array $paths A path or an array of paths where to look for resources
+     */
+    public function __construct($paths = [])
+    {
+        $this->paths = (array) $paths;
+    }
+
+    /**
+     * Returns a new Finder object with the resource paths set.
+     *
+     * @param string $path An optional path
+     *
+     * @return Finder The Finder object
+     */
+    public function in($path = null)
```

Comment:
You are right. I will rename the method to `find()` and add a `findIn()` method, which you can pass a subfolder as argument.